src/main/java/org/snomed/snowstorm/core/data/services/MultiSearchService.java

```diff
@@ -68,26 +73,37 @@ public class MultiSearchService implements CommitListener {
 	private MultiBranchCriteria cachedBranchCriteria = null;
 	private LocalDate cacheDate = null;
 
-	public Page<Description> findDescriptions(DescriptionCriteria criteria, String ecl, PageRequest pageRequest) {
-
-		if (ecl != null) {
+	public Page<Description> findDescriptions(DescriptionCriteria criteria, Boolean refsetsOnly, PageRequest pageRequest) {
+	
+		//Handle null parameter
+		if(refsetsOnly == null) {
+			refsetsOnly = false;
+		}
+		
+		if (refsetsOnly) {
+			// Grab all published branches and also all "REFSETS" branches, 
+			// and do the "<446609009" ECL call on all of them to get the refset concepts
 			// ECL -> conceptIds
-			BranchCriteria branchCriteria = versionControlHelper.getBranchCriteria("MAIN");
-			//MultiBranchCriteria branchesQuery = getBranchesQuery();
-			Page<Long> page = eclQueryService.selectConceptIds(ecl, branchCriteria, true, null, null);
-			criteria.conceptIds(page.getContent());
+			MultiBranchCriteria branchesQuery = getBranchesQueryWithRefsets();
+			Set<Long> conceptIds = new HashSet<>();
+			for(BranchCriteria branchCriteria : branchesQuery.getBranchCriteria())
+			{
+				conceptIds.addAll(eclQueryService.selectConceptIds("<446609009", branchCriteria, true, null, null).getContent());
```

Comment:
Please add this SCTID to org.snomed.snowstorm.core.data.domain.Concepts and reference as a String constant.

Reply:
Ah, I hadn't discovered the Concepts section yet. And actually, it looks like it's already in there:

`public static final String REFSET_SIMPLE = "446609009";`

Will update this section of code to reference it.

Reply:
Updates made and changes pushed.